pkg/visor/init_windows.go

```diff
@@ -12,3 +12,7 @@ func initDmsgpty(ctx context.Context, log *logging.Logger) error {
 	log.Error("dmsgpty is not supported on windows.")
 	return nil
 }
+
+func appendExecPlatform(envs []string) {
```

Comment:
What is this function exactly? I didn't see used anywhere. (Same as `init_windows.go` in `skywire-cli` command visor)

Reply:
good catch, I'll delete it. Thanks man